Cargo.toml

```diff
@@ -12,6 +12,7 @@ authors = ["C2Rust"]
 version = "0.0.0"
 publish = false
 edition = "2021"
+rust-version = "1.82" # MSRV
```

Comment:
do we need this? for usage in the `bzip2` crate it might be nice to have a substantially lower msrv?

Reply:
The CI config expects some MSRV. I just picked the latest rustc as that is what I'm locally developing on. The actual MSRV is at least 1.77 though for the binaries as I used `c"foo"` C string literals.

Reply:
We can always rewrite code that uses newer features again in the future if necessary.

Reply:
the `c"foo"` literals can mostly go as we clean up the code I think? at least that has been the case in `bzlib.rs`. but I'm fine with picking this now and lowering it later when inclusion in `bzip2` becomes more concrete.